ui/packages/catalog-ui-search/src/main/webapp/js/widgets/openlayers.bbox.js

```diff
@@ -75,12 +75,29 @@ Draw.BboxView = Marionette.View.extend({
     let west = parseFloat(model.get('mapWest'))
 
     if (isNaN(north) || isNaN(south) || isNaN(east) || isNaN(west)) {
```

Comment:
```suggestion
    if (isNaN(north) || isNaN(south) || isNaN(east) || isNaN(west) || south > north) {
```

Should be able to add that condition and get rid of the if statement below